lib/modules/manager/index.ts

```diff
@@ -10,21 +12,24 @@ import type {
   Result,
 } from './types';
 export { hashMap } from './fingerprint.generated';
-const managerList = Array.from(managers.keys());
+const managerList = Array.from(managers.keys()); // does not include custom managers
+const customManagerList = getCustomManagerList();
```

Comment:
```suggestion
const customManagerList = Array.from(custommManagers.keys());
```
simplify 😉

probably inline into array below, then you can skip the array.from and we safe some memory